index.js

```diff
@@ -9,27 +9,6 @@ const spawn = require('cross-spawn');
 const chalk = require('chalk');
 const which = require('which');
 
-const exitOnError = async function(promise) {
-    try {
-        await promise;
-    } catch (e) {
-        console.log('Error: ', e);
-        process.exit(1);
-    }
-};
```

Comment:
I removed this function, since I believe this is the default behavior. If JavaScript encounters an error, it stops executing and logs the error.